upper-bounds-check/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud</groupId>
+  <artifactId>upper-bounds-check</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Cloud Platform Supported Libraries Upper Bounds Check</name>
+  <description>
+    Import the Google Cloud BOM in a simple project and verify that each library imported is the
+    most recent of the versions available in the dependency tree. This check is similar to the
+    Library BOM's check:
+    https://github.com/GoogleCloudPlatform/cloud-opensource-java/blob/master/boms/upper-bounds-check/pom.xml
+  </description>
+
+  <organization>
+    <name>Google LLC</name>
+    <url>https://cloud.google.com</url>
+  </organization>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-bom</artifactId>
+        <version>0.153.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-bom:current} -->
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-pubsub</artifactId>
```

Comment:
google-cloud-core is not part of the google-cloud-bom. Therefore I picked pubsub (just a random artifact that I sometimes see upper-bound failure)

Reply:
Does the ordering here matter? If google-cloud-core is not included in the google-cloud-bom, then the ordering of the maven dependency declarations might matter for whether or not this test passes.

Reply:
Yes, the ordering matters. Sometimes we _fix_ the check failure by switching artifacts. I don't like it.